src/utils/fnr-utils.ts

```diff
@@ -1,4 +1,40 @@
-export function erGydligishFnr(fnr: string) {
+export function erGyldigishFnr(fnr: string) {
     const numeric = !/\D/.test(fnr);
     return fnr.length === 11 && numeric;
 }
+
+const kontrollRekke1 = [3, 7, 6, 1, 8, 9, 4, 5, 2];
+const kontrollRekke2 = [5, 4, 3, 2, 7, 6, 5, 4, 3, 2];
+function kontrollSiffer(ident: number[], kontrollrekke: number[]): number {
+    let sum = 0;
+    for (let i = 0; i < Math.min(ident.length, kontrollrekke.length); i++) {
+        sum += ident[i] * kontrollrekke[i];
+    }
+    const kontroll = sum % 11;
+    return kontroll !== 0 ? 11 - kontroll : 0;
+}
```

Comment:
Prøver å lese meg opp på matten her. Er det siste `1`-tallet droppet fra kontrollrekkene for å unngå at vi får `kontrollnr === 10`?
http://www.fnrinfo.no/Teknisk/KontrollsifferSjekk.aspx

Reply:
Det følger nok bare en litt annen algoritme; https://no.wikipedia.org/wiki/F%C3%B8dselsnummer 
e.g `k1 = 11 - ((3 × d1 + 7 × d2 + 6 × m1 + 1 × m2 + 8 × å1 + 9 × å2 + 4 × i1 + 5 × i2 + 2 × i3) mod 11),`

Forskjellen er hvorvidt man legger til tiende (for kontrollrekke 1) sifferet til `sum` og sjekker `kontroll == 0`, eller om man sjekker at `11 - (kontroll % 11) == ident[9]`. Så for de som husker modulær aritmetikken bedre enn hva jeg gjør kan sikkert bevise dette. :D